src/client/components/menu/Menu.component.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
+import './Menu.styles.css';
+import faUser from '../../assets/images/user-login.png';
+import faHeart from '../../assets/images/favorite-icon.png';
+import faShoppingCart from '../../assets/images/shopping-cart.png';
+
+export const Menu = () => {
+  return (
+    <menu>
+      <header>
+        <BrowserRouter>
+          <div className="logo_container">
+            <div className="user-icon">
+              <div className="dropdown">
+                <img src={faUser} alt="login" />
+
+                <div id="login" className="dropdown-content">
+                  <Link className="text-link" to="/login">
+                    LOGIN / SIGNUP
+                  </Link>
+                </div>
+              </div>
+            </div>
+            <div className="icons">
+              <img src={faHeart} alt="favorite" />
```

Comment:
You can add the `className="icons"` on line 26 and 29, and remove the containers you currently have around each of the images, **or** have both images under the same div with the `className="icons"`